knotx-knot/knotx-knot-handlebars/src/main/java/io/knotx/knot/templating/impl/HandlebarsKnotProxyImpl.java

```diff
@@ -69,6 +92,41 @@ public HandlebarsKnotProxyImpl(HandlebarsKnotConfiguration configuration) {
     });
   }
 
+  private Template template(Fragment fragment) {
+    try {
+      byte[] cacheKeyBytes = digest.digest(fragment.content().getBytes(StandardCharsets.UTF_8));
```

Comment:
to make it more readable, instead of those two lines, extract it into a method, e.g.:
```
String cacheKeyForFragment(Fragment fragment)
```

Reply:
fixed